Structure_oM/MaterialFragments/LaminatedVeneerLumberParallel.cs

```diff
@@ -0,0 +1,202 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using BH.oM.Geometry;
+using BH.oM.Quantities.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.oM.Structure.MaterialFragments
+{
+
+    [Description("Structural timber material of type Laminated Veneer Lumber without crossband veneers. To be used on structural elements and properties, or as a fragment of the physical material.\n" +
+                 "Generally only applicable for use in beam elements.\n" +
+                 "Note: Properties for LVL are not part of a harmonised standard and therefore vary between manufacturers and products.")]
+    public class LaminatedVeneerLumberParallel : BHoMObject, ITimber
+    {
+        /***************************************************/
+        /**** Properties - General and analysis         ****/
+        /***************************************************/
+
+        [Description("Name. A unique name is required for some structural packages to create and identify the object.")]
+        public override string Name { get; set; }
```

Comment:
```suggestion
        [Description("A unique name is required for some structural packages to create and identify the object.")]
        public override string Name { get; set; }
```
Similar as comments to LVLCross.